app/services/foreman_seeder.rb

```diff
@@ -27,8 +28,12 @@ def plugin_seeds
     end.flatten.compact
   end
 
+  def templates
+    SeedHelper.report_templates + SeedHelper.provisioning_templates + SeedHelper.partition_tables_templates
```

Comment:
Can there be other types of templates (e.g. some that plugins define).

Reply:
yes. There are in REX for example. I think in that case the plugin can override this method to add them to the list.

Reply:
Problem is, plugin authors won't. This is not a good design, we should add some registration API mechanism to prevent these kind of errors. But since this is a bugfix, I am fine merging that - we should fix this later.